cmd/goal/network.go

```diff
@@ -112,6 +128,18 @@ var networkCreateCmd = &cobra.Command{
 			reportErrorf(infoNetworkAlreadyExists, networkRootDir)
 		}
 
+		// If genesisDir is specified, copy files over
```

Comment:
Thats pretty cool, it just worked?

We should create some followup issues to make it slightly more lenient. Currently if you copy over `Wallet1.0.30000000.partkey` it would be ignored in favor of regenerating `Wallet1.0.30000.partkey`.

Reply:
Adding a `--round-override` option to the pregen command might also be convenient.

Reply:
> We should create some followup issues to make it slightly more lenient. Currently if you copy over Wallet1.0.30000000.partkey it would be ignored in favor of regenerating Wallet1.0.30000.partkey.

That's an interesting idea. For this PR, the command needs to have the template files match (have the same first/last part key rounds and matching wallet data), or else it is regenerated like you have noticed

Reply:
Let's save this for a future PR. I like how light a touch you were able to use here by using the old/unused functionality